src/components/Carousel.tsx

```diff
@@ -882,7 +884,7 @@ export default class Carousel extends React.Component<Props, State> {
                 <div className={klass.CAROUSEL(true)} style={{ width: this.props.width }}>
                     {this.props.renderArrowPrev(this.onClickPrev, hasPrev, this.props.labels.leftArrow)}
                     <div className={klass.WRAPPER(true, this.props.axis)} style={containerStyles}>
-                        {this.props.swipeable ? (
+                        {isSwipeable ? (
```

Comment:
I suspect you will need to do the same on the Thumbs component...

Reply:
Agreed.